.github/workflows/e2e-fulltests-ci.yml

```
@@ -109,34 +150,45 @@ jobs:
       - name: ci/generate-test-variables
         id: generate
         run: |
-          COMMIT_SHA="${{needs.resolve-ref.outputs.commit_sha}}"
           # BUILD_ID format: $pipelineID-$imageTag-$testType-$serverType-$serverEdition
           # Reference on BUILD_ID parsing: https://github.com/saturninoabril/automation-dashboard/blob/175891781bf1072c162c58c6ec0abfc5bcb3520e/lib/common_utils.ts#L3-L23
-          BUILD_ID_PREFIX="${{ github.run_id }}_${{ github.run_attempt }}-${COMMIT_SHA::7}"
+          BUILD_ID_PREFIX="${{ github.run_id }}_${{ github.run_attempt }}-${SERVER_IMAGE_TAG}"
```

Should this be with a `-` instead of `_` ? 
```suggestion
          BUILD_ID_PREFIX="${{ github.run_id }}-${{ github.run_attempt }}-${SERVER_IMAGE_TAG}"
```

---

Great eye Antonis :eagle:  
But the `_` is actually intentional: the Automation Dashboard uses the dash to split the fields of the `BUILD_ID` value, and for the first field (`pipelineID`) to be unique, we need to have both the Run ID _and_ the run attempt in it.
I have documented the `BUILD_ID` format two lines above.